features/FEATURE_BLE/targets/TARGET_CORDIO_LL/stack_adaptation/fake_lhci_drv.h

```diff
@@ -21,27 +21,38 @@
 extern "C" {
 #endif
 
+/**
+ * Provide this callback in your HCI driver.
+ *
+ * @param prot Must be CHCI_TR_PROT_BLE.
+ * @param hci_type HCI type, HCI_ACL_TYPE or HCI_CMD_TYPE.
+ * @param len Buffer length.
+ * @param pData Data to be sent.
+ * @return Number of bytes processed.
+ */
+uint16_t controllerToHostWrite(uint8_t prot, uint8_t hci_type, uint16_t len, uint8_t *pData);
```

Comment:
Should we add an `mbed_` prefix ? For the function name it might be better to use `PascalCase` as it is used in the cordio stack or `snake_case` that is used within mbed. `camelCase` is an anomaly of the BLE API.

Reply:
sorry, it's just a typo not a planned subversion of our code style